### PR TITLE
[ISS-77] implement django revers url delete hard code html && js

### LINKED ISSUE #77: implement django revers url - delete hard code html && js

### PR DESCRIPTION
**Pull request by Hamilkar247**
fixes #77

### CODE REVIEW

**Comment by Hamilkar247** [comment]
to merge after #42

**Comment by jlatusek** [comment]
Use [ISS-77] in head  of pull request

**Comment by Hamilkar247** [comment]
ready to merge

**Comment by jlatusek** [comment]
You don't have to write ready to merge just remove draft label, and now I can't merge yet because of that label

File: front/templates/front/decks.html [hunk 1]
```diff
@@ -48,6 +48,14 @@ <h5>Title description, Sep 2, 2017</h5>
   {% if user.is_authenticated %}
     {{ block.super }}
     <!-- scripts - decks.html -->
+    <script>
+      function generate_api_v1_deck(){
```
Comment (jlatusek) [inline front/templates/front/decks.html:52]: I think it is fine solution, but make it more generic and extract it to another file, so you could use this in another view.

**Review by jlatusek — CHANGES_REQUESTED** [review]

File: front/templates/front/decks.html (hunk 1 above)
Comment (Hamilkar247) [reply front/templates/front/decks.html]: ok i do it in other way

**Comment by Hamilkar247** [comment]
I do a little changes in deck.html and decks.js - look on them

**Comment by Hamilkar247** [comment]
@jlatusek

**Comment by Hamilkar247** [comment]
ok i hope this final final final version

**Comment by Hamilkar247** [comment]
@jlatusek

**Pull request merged**